src/main/java/converter/DataConverterImpl.java

```diff
@@ -0,0 +1,19 @@
+package converter;
+
+import java.util.List;
+import model.FruitTransaction;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> converterToTransaction(List<String> data) {
+        return data.stream()
+                .skip(1) // Skip header
+                .map(line -> {
+                    String[] parts = line.split(",");
+                    FruitTransaction.Operation operation
+                            = FruitTransaction.Operation.fromCode(parts[0]);
+                    return new FruitTransaction(operation, parts[1], Integer.parseInt(parts[2]));
+                })
+                .toList();
+    }
+}
```

Comment:
The class `DataConverterImpl` should be placed in the `service` or `service.impl` package according to the project structure requirements. Make sure you move it to the appropriate package to maintain a clean code structure.